extensions/terminal-suggest/src/completions/code.ts

```diff
@@ -342,6 +412,394 @@ export function parseInstalledExtensions(out: string): Fig.Suggestion[] | undefi
 	return extensions;
 }
 
+export const commonAuthOptions: Fig.Option[] = [
+	{
+		name: '--access-token',
+		description: 'An access token to store for authentication',
+		isRepeatable: true,
+		args: {
+			name: 'access_token',
+			isOptional: true,
+		},
+	},
+	{
+		name: '--refresh-token',
+		description: 'An access token to store for authentication',
+		isRepeatable: true,
+		args: {
+			name: 'refresh_token',
+			isOptional: true,
+		},
+	},
+	{
+		name: '--provider',
+		description: 'The auth provider to use. If not provided, a prompt will be shown',
+		isRepeatable: true,
+		args: {
+			name: 'provider',
+			isOptional: true,
+			suggestions: [
+				'microsoft',
+				'github',
+			],
+		},
+	}
+];
+
+const tunnelSubcommands: Fig.Subcommand[] = [
+	{
+		name: 'prune',
```

Comment:
Inlining this subcommand would make reading it a little less confusing as tunnel can refer to multiple things (`codeTunnelSubcommands` vs `tunnelSubcommands`).